### PR TITLE
Add libraries for CS199 Data Science w/ Python external grader

### PR DESCRIPTION
**Pull request by nicknytko**
Adding:

- `scikit-image`
- `scikit-learn`
- `requests`
- `folium`
- `colormath`
- `nltk`
- `beautifulsoup4`
- `bokeh`

### CODE REVIEW

File: graders/python/Dockerfile [hunk 1]
```diff
@@ -10,6 +10,6 @@ RUN yum -y update \
     && yum install -y python36u python36u-devel python36u-pip \
     && yum install -y graphviz graphviz-devel \
     && ln -s /usr/bin/python3.6 /usr/bin/python3 \
-    && python3 -m pip install --no-cache-dir -r /requirements.txt
+     ; python3 -m pip install --no-cache-dir -r /requirements.txt
```
Comment (mwest1066) [inline graders/python/Dockerfile]: What's the reason for this change?

File: ChangeLog.md [hunk 2]
```diff
@@ -48,6 +48,8 @@
   * Add support for rendering graphs using adjacency matrices in `pl-graph` (Nicolas Nytko).
 
   * Add `gvsu.edu` as an institution option (Matt West).
+  
+  * Add `scikit-learn`, `scikit-image`, and other data science libraries to external grader (Nicolas Nytko).
```
Comment (mwest1066) [inline ChangeLog.md]:
```suggestion
  * Add `scikit-learn`, `scikit-image`, and other data science libraries to external Python grader (Nicolas Nytko).
```

File: graders/python/Dockerfile (hunk 1 above)
Comment (nicknytko) [reply graders/python/Dockerfile]: When I was building the docker image locally on my machine it was error-ing out because python3 already existed, so I made it fail silently if it can't create the symlink

Comment (mwest1066) [reply graders/python/Dockerfile]: Good point -- I think the right fix here is to apply the same `python3` update as @trombonekenny did for `plbase`: https://github.com/PrairieLearn/PrairieLearn/pull/1719
